articles/frontdoor/front-door-faq.yml

```
@@ -262,7 +262,8 @@ sections:
       - question: |
           Can I set alerts with Azure Front Door?
         answer:  |
-          Yes, Azure Front Door does support alerts. Alerts are configured on metrics. 
+          Yes, Azure Front Door does support alerts. Alerts are configured on metrics. For reference to create alerts, please check https://docs.microsoft.com/en-us/azure/frontdoor/standard-premium/how-to-monitor-metrics#configure-alerts-in-azure-portal 
+          And for more information about metrics available on Front Door diagnostic monitoring refer to https://docs.microsoft.com/en-us/azure/frontdoor/front-door-diagnostics#metrics
```

```suggestion
          Yes, Azure Front Door does support alerts. Alerts are configured on metrics. For reference to create alerts, please check [Configure alerts](/azure/frontdoor/standard-premium/how-to-monitor-metrics#configure-alerts-in-azure-portal).
          And for more information about metrics available on Front Door diagnostic monitoring refer to [Front Door metrics](/azure/frontdoor/front-door-diagnostics#metrics).
```
fixing links according to MS style and Doc criteria